rules-reviewed/jboss-eap/websphere/tests/websphere-mq.windup.test.xml

```diff
@@ -0,0 +1,500 @@
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.Session`."/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-57000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.QueueSession`."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.QueueSession`."/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-58000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.TopicSession`."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.TopicSession`."/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-59000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XASession`."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XASession`."/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-60000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XAQueueSession`."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XAQueueSession`."/>
+                </perform>
+            </rule>
+            <rule id="websphere-mq-61000-test-h1">
+                <when>
+                    <not>
+                        <hint-exists message="A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XATopicSession`."/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Hint not found: A more general, and recommended, approach is to use the Java EE 6 JMS class `javax.jms.XATopicSession`."/>
+                </perform>
+            </rule>
+
+
+
+            <!-- Covers these classes:
```

Comment:
Is this comment necessary when the rule-id is now specified in the test rule-id? Please remove.